x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/event_correlation_rule.cy.ts

```diff
@@ -174,4 +183,34 @@ describe('EQL rules', { tags: ['@ess', '@serverless'] }, () => {
         });
     });
   });
+
+  describe('with source data requiring EQL overrides', () => {
+    before(() => {
+      cy.task('esArchiverLoad', { archiveName: 'no_at_timestamp_field', type: 'ftr' });
+    });
+
+    after(() => {
+      cy.task('esArchiverUnload', { archiveName: 'no_at_timestamp_field', type: 'ftr' });
+    });
+
+    it('includes EQL options in query validation', () => {
```

Comment:
[Of course](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/5526). 

Do you see anything in the test itself that looks suspect?

Reply:
nothing suspicious, but when adding new test we usually run them in flaky test runner to ensure they are not flaky, as an effort to reduce overall Cypress tests flakiness.
Since even perfectly written code does not guarantee tests won't be flaky unfortunately